orte/mca/rmaps/base/rmaps_base_map_job.c

```diff
@@ -587,25 +587,23 @@ void orte_rmaps_base_display_map(orte_job_t *jdata)
                     continue;
                 }
                 procbitmap = NULL;
-                if (orte_get_attribute(&proc->attributes, ORTE_PROC_CPU_BITMAP, (void**)&procbitmap, OPAL_STRING) &&
-                    NULL != procbitmap) {
-                    locality = opal_hwloc_base_get_relative_locality(node->topology->topo,
-                                                                     p0bitmap,
-                                                                     procbitmap);
-                    opal_output(orte_clean_output, "\t\t<rank=%s rank=%s locality=%s>",
-                                ORTE_VPID_PRINT(p0->name.vpid),
-                                ORTE_VPID_PRINT(proc->name.vpid),
-                                opal_hwloc_base_print_locality(locality));
+                orte_get_attribute(&proc->attributes, ORTE_PROC_CPU_BITMAP, (void**)&procbitmap, OPAL_STRING);
```

Comment:
This isn't safe - while it generally is true that all procs will be bound if the first one is, there is a chance that the proc isn't bound and therefore has no bitmap attribute. Is there some reason to remove the "if"?

Reply:
Hm, I'll confirm with @markalle , but I don't have an objection with putting it back.

Reply:
@rhc54 done.